radis/test/lbl/test_loader.py

```diff
@@ -222,13 +222,175 @@ def test_custom_abundance(verbose=True, plot=False, *args, **kwargs):
     assert s.compare_with((3 * s3.take("abscoeff")), "abscoeff", rtol=0.5e-2, plot=True)
 
 
+def test_vaex_and_pandas_dataframe_fetch_databank():
```

Comment:
Same as above, please write :  
- what are you testing 
- what results do you expect

Reply:
I will  add the comments